tests/wpt/web-platform-tests/css/cssom/null-owner.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<head>
+  <meta charset="utf-8">
+  <title>ownerNode null test</title>
+  <link rel="help" href="https://drafts.csswg.org/cssom/#dom-stylesheet-ownernode">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+  <style></style>
+</head>
+<body>
+  <script>
+  	test(function() {
+      let sheet = document.querySelector("style").sheet;
```

Comment:
It depends on you but, personally, I'd use the same way as @emilio said.

```js
let styleElement = document.querySelector("style");
let sheet = styleElement.sheet;
```

Reply:
https://github.com/servo/servo/pull/23096#discussion_r268901706
Initially this is what emilio suggested. I had added styleElelemt just because I thought I needed it.